packages/components/src/components/DistributionChart.tsx

```diff
@@ -44,15 +45,62 @@ export const DistributionChart: React.FC<DistributionChartProps> = ({
   showControls,
   logX,
   expY,
+  truncateToNthci,
 }) => {
   const [isLogX, setLogX] = React.useState(logX);
   const [isExpY, setExpY] = React.useState(expY);
 
   React.useEffect(() => setLogX(logX), [logX]);
   React.useEffect(() => setExpY(expY), [expY]);
 
-  const shape = distribution.pointSet();
   const [sized] = useSize((size) => {
+    const delta = truncateToNthci / 100 / 2;
```

Comment:
It's weird to me that this has to do operations in the case where we don't want truncation.